quinn/src/platform/unix.rs

```diff
@@ -373,3 +399,40 @@ pub const BATCH_SIZE: usize = 32;
 
 #[cfg(any(target_os = "macos", target_os = "ios"))]
 pub const BATCH_SIZE: usize = 1;
+
+#[cfg(target_os = "linux")]
+lazy_static! {
+    static ref CAPABILITIES: super::UdpCapabilities = {
+        /// Checks whether GSO support is available by setting the UDP_SEGMENT
+        // option on a socket.
```

Comment:
I guess you meant to continue the docstring here? Let's remove the period at the end.

Reply:
(Only relevant if we keep the inline function.)